content/en/agent/logs/log-transport.md

```diff
@@ -0,0 +1,139 @@
+further_reading:
+- link: "agent/logs/advanced_log_collection/#filter-logs"
+  tag: "Documentation"
+  text: "Filter logs sent to Datadog"
+- link: "agent/logs/advanced_log_collection/#scrub-sensitive-data-from-your-logs"
+  tag: "Documentation"
+  text: "Scrub sensitive data from your logs"
+- link: "agent/logs/advanced_log_collection/#multi-line-aggregation"
+  tag: "Documentation"
+  text: "Multi-line log aggregation"
+- link: "agent/logs/advanced_log_collection/#tail-directories-by-using-wildcards"
+  tag: "Documentation"
+  text: "Tail directories by using wildcards"
+- link: "agent/logs/advanced_log_collection/#global-processing-rules"
+  tag: "Documentation"
+  text: "Global processing rules"
+---
+
+
+## Default agent behavior
+
+Since Agent v6.19+/v7.19+, the default transport is HTTPS with compression instead of TCP.
+At agent start, if log collection is enabled, the agent runs a HTTPS connectivity test:
+
+* if successful, then the agent uses compressed HTTPS transport
+* otherwise the agent fallbacks to TCP transport
+
+This connectivity test mechanism is only running at Agent startup. If the connectivity is failing for both HTTPS and TCP at Agent startup time, then the Agent uses TCP transport until its next restart (and then rerun a new connectivity test)
+
+To check which transport is used by the agent, run the [agent statuts command][1].
+
+{{< img src="agent/logs/agent-status.png" alt="Agent status"  style="width:70%;">}}
+
+For older version of the agent, TCP is used by default. We strongly recommend you to enforce HTTPS transport if you are running v6.14+/v7.14+ and HTTPS transport and compression if you are running v6.16+/v7.16+. 
+
+
+## Enforce a specific transport
+
+Enforce the use of TCP or HTTPS transport by using the following configurations.
+
+{{< tabs >}}
+{{% tab "HTTP" %}}
+
+To enforce HTTP transport, update the Agent's [main configuration file][1] (`datadog.yaml`) with:
+
+```yaml
+logs_enabled: true
+logs_config:
+  use_http: true
+```
+
+To send logs with environment variables, configure the following:
+
+* `DD_LOGS_ENABLED`
+* `DD_LOGS_CONFIG_USE_HTTP`
+
+Note: HTTPS transport is supported on Datadog Agent version 6.14+/7.14+
+
+[1]: /agent/guide/agent-configuration-files
+{{% /tab %}}
+{{% tab "TCP" %}}
+
+To enforce TCP transport, update the Agent's [main configuration file][1] (`datadog.yaml`) with:
+
+```yaml
+logs_enabled: true
+logs_config:
+  use_tcp: true
+```
+To send logs with environment variables, configure the following:
+
+* `DD_LOGS_ENABLED`
+* `DD_LOGS_CONFIG_USE_TCP`
+
+By default, the Datadog Agent sends its logs to Datadog over TLS-encrypted TCP. This requires outbound communication (on port `10516` for US site and port `443`for EU site).
+
+[1]: /agent/guide/agent-configuration-files
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Note: Setting up a [SOCKS5 proxy][4] server will also enforce TCP transport because socks5 proxies are not yet supported in HTTPS with compression.
```

Comment:
```suggestion
**Note**: Setting up a [SOCKS5 proxy][4] server enforces TCP transport because socks5 proxies are not yet supported in HTTPS with compression.
```